cmd/fleet/handleStatus.go

```diff
@@ -0,0 +1,43 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package fleet
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/elastic/elastic-agent-client/v7/pkg/proto"
+	"github.com/julienschmidt/httprouter"
+	"github.com/rs/zerolog/log"
+)
+
+func (rt Router) handleStatus(w http.ResponseWriter, _ *http.Request, _ httprouter.Params) {
+	status := rt.sm.Status()
```

Comment:
This appears to be unauthorized if I am reading this correctly.  We should avoid leaking the fleet server version in an unauthorized API. It gives an attacker knowledge about which software is executing, which is problematic if we have a vulnerability somewhere.

Is this version necessary here?

Reply:
No its not, we can remove it. But I think Kibana expose version unauthorized as well.

Reply:
We should remove.  Kibana shouldn't be doing that either.  

FWIW, we had the same issue at Endgame, the web app was designed to expose too much pre-auth that provided a reliable signature for an attacker.